src/app/components/Footer/index.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { arrayOf, shape, string, node, bool } from 'prop-types';
+import { C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+import Link from './Link';
+import List from './List';
+import { AmpCookieSettingsButton } from '#app/containers/ConsentBanner/Banner/cookie.amp';
+
+const SitewideLinksWrapper = styled.div`
+  ${({ script }) => script && getBrevier(script)}
+  ${({ service }) => getSansRegular(service)}
+  background-color: ${C_EBON};
+
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_BELOW_400PX};
+  }
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const ConstrainedWrapper = styled.div`
+  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  margin: 0 auto;
+  ${({ trustProjectLink }) =>
+    trustProjectLink && `padding-top: ${GEL_SPACING};`}
+`;
+
+const StyledParagraph = styled.p`
+  color: ${C_WHITE};
+  margin: 0;
+  padding: ${GEL_SPACING_DBL} 0;
+`;
+
+const StyledAmpCookieSettingsButton = styled(AmpCookieSettingsButton)`
+  background: none;
+  border: none;
+  color: ${C_WHITE};
+  cursor: pointer;
+  display: block;
+  font-weight: 700; /* Used instead of Reith Sans Bold since it is not worth the performance cost in this case. */
```

Comment:
Perhaps naive question, why do we have a bold font at all if this is an option? Just wondering why it's okay here but not in other places? 
Has this been taken from the existing code ported from psammead here?
https://github.com/bbc/simorgh/pull/8929/files#diff-510c3d49b2b2434f9aa3397807021e060b374bf2e88743373811f3c7e4dadcfcR7

Reply:
Yes, good catch - we should be using the sans bold utility here. I will update this.